platforms/core-configuration/kotlin-dsl-tooling-builders/src/main/kotlin/org/gradle/kotlin/dsl/tooling/builders/KotlinDslScriptsModelBuilder.kt

```diff
@@ -160,7 +160,7 @@ abstract class AbstractKotlinDslScriptsModelBuilder : ToolingModelBuilder {
     private
     fun requireRootProject(project: Project) =
         require(project == project.rootProject) {
-            "$MODEL_NAME can only be requested on the root project, got '$project'"
+            "$MODEL_NAME can only be requested on the root project, got $project"
```

Comment:
ℹ️  We do not seem to test against this string anywhere, because I guess it's not very relevant.
The change here is because the current string would produce this kind of output if ever triggered:
```
<MODEL_NAME> can only be requested on the root project, got 'project ':a''
```